client/my-sites/site-settings/form-seo.jsx

```diff
@@ -32,7 +32,7 @@ import FormFieldset from 'components/forms/form-fieldset';
 import FormLabel from 'components/forms/form-label';
 import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import CountedTextarea from 'components/forms/counted-textarea';
-import SeoSettingsUpgradeNudge from 'components/seo/settings-upgrade-nudge';
+import SeoSettingsUpgradeNudge from 'my-sites/upgrade-nudge/expanded';
```

Comment:
Cool. Also, please call it `upgrade-nudge-expanded` as per https://github.com/Automattic/wp-calypso/blob/master/docs/components.md, sub-folder component should not care about its parent naming (and thus could be moved anywhere anytime without breaking stuff).